deploy/examples/operator-openshift.yaml

```diff
@@ -682,7 +682,7 @@ spec:
             - name: ROOK_CURRENT_NAMESPACE_ONLY
               value: "false"
 
-            # Whether to start pods as privileged that mount a host path, which includes the Ceph mon and osd pods.
+            # Whether to start pods as privileged that mount a host path, which includes the Ceph mon, osd pods and csi provisioners(if logrotation is on).
```

Comment:
IMO, I would not bother with this change since CSI is going to be its own operator soon.